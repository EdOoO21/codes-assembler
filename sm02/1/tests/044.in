65 109 436 765 -762 -639 -223 -539 -407 -941 -540 -762 164 -982 -795 -680 586 52 410 -380 420 298 -225 210 -171 568 415 372 -333 -267 -850 -307 723 -421 264 -675 -771 46 -857 205 -807 -448 963 -355 -780 931 280 683 985 -407 -929 -295
