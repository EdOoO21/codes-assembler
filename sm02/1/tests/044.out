-295
-929
-407
985
683
280
931
-780
-355
963
-448
-807
205
-857
46
-771
-675
264
-421
723
-307
-850
-267
-333
372
415
568
-171
210
-225
298
420
-380
410
52
586
-680
-795
-982
164
-762
-540
-941
-407
-539
-223
-639
-762
765
436
109
65
